docs/sources/flow/reference/config-blocks/import.file.md

```diff
@@ -15,16 +15,20 @@ title: import.file
 
 {{< docs/shared lookup="flow/stability/beta.md" source="agent" version="<AGENT_VERSION>" >}}
 
-The `import.file` block imports custom components from a file and exposes them to the importer.
+The `import.file` block imports custom components from a file or a folder and exposes them to the importer.
 `import.file` blocks must be given a label that determines the namespace where custom components are exposed.
 
+Imported folders are treated as single modules to support composability.
+That means that you can define a custom component in one file and use it in another custom component in another file
+in the same folder.
+
 [module]: {{< relref "../../concepts/modules.md" >}}
 
 ## Usage
 
 ```river
 import.file "NAMESPACE" {
-  filename = FILENAME
+  filename = FILENAME_OR_FOLDERNAME
```

Comment:
nit: the documentation for the [run command](https://grafana.com/docs/agent/latest/flow/reference/cli/run/) calls this `PATH_NAME`; should we use the same here to make it consistent?